include/circt/Dialect/SV/SVStatements.td

```diff
@@ -879,3 +882,167 @@ def MacroDefOp : SVOp<"macro.def",
     MacroDeclOp getReferencedMacro(const hw::HWSymbolCache *cache);
   }];
 }
+
+
+//===----------------------------------------------------------------------===//
+// Function/Call
+//===----------------------------------------------------------------------===//
+
+def FuncOp : SVOp<"func",
+     [IsolatedFromAbove, Symbol, OpAsmOpInterface, ProceduralRegion,
+      DeclareOpInterfaceMethods<HWModuleLike>,
+      DeclareOpInterfaceMethods<PortList>,
+      FunctionOpInterface, HasParent<"mlir::ModuleOp">]> {
+  let summary = "A System Verilog function";
+  let description = [{
+    `sv.func` reresents System Verilog function in SV spec 13.4.
```

Comment:
```suggestion
    `sv.func` reresents SystemVerilog function in SV spec 13.4.
```

Can you also mention which exact version of the spec this is? They tend to insert paragraphs, which makes the numbering unstable. Something like `see IEEE 1800-2017 section 13.4 "section title"` maybe?